### PR TITLE
Loops_RobertProdanov

### PR DESCRIPTION
**Pull request by RobertProdanov03**

### CODE REVIEW

File: Loops_RobertProdanov/.gitignore
Comment (iivanov00) [inline Loops_RobertProdanov/.gitignore:1]: you should not commit your gitignore and idea folder

**Review by iivanov00 — APPROVED** [review]

**Comment by RobertProdanov03** [comment]
With which file I have a problem?

**Comment by iivanov00** [comment]
> With which file I have a problem?

I don't know you should see this, when you merge develop in this branch

**Comment by RobertProdanov03** [comment]
I can't see conflict on my side, even github let me make merge

**Comment by RobertProdanov03** [comment]
Yes, i did

**Comment by RobertProdanov03** [comment]
I updated from main again. Maybe the conflict problem is solved

**Review by iivanov00 — APPROVED** [review]

**Comment by iivanov00** [comment]
There are still conflicts maybe you should remove the package that is not yours(Loops_Petko..) from the commit/PR

**Comment by RobertProdanov03** [comment]
I will make new branch

**Pull request closed without merging**